release-team/role-handbooks/release-signal/project-boards.md

```diff
@@ -0,0 +1,150 @@
+current status of all issues and PRs targeting the release, their priorities, and to distribute the work amongst the team.
+
+## Project Board Overview
+
+### CI Signal Project Board
+
+The CI component of the team is using [GitHub project board](https://github.com/orgs/kubernetes/projects/68) to track the
+currently failing and flaking tests across various testgrid dashboards.
+
+Release Signal team members are expected to observe the testgrid dashboards and create issues, and add them to the GitHub project
+board. You may create issues from the board or add them later. This board acts as a single point of reference for the current
+state of the CI and is referred to by both the general public and the release management team. Issues directly created in the
+kubernetes/kubernetes repository are not directly added to the board; they need to be manually added.
+
+The CI component of the release signal team only looks at the dashboard mentioned in this document, failures on other boards, if
+important and not mentioned may go unnoticed by the team.
+
+#### Information on the CI Signal Board
+
+The release signal team members set the following items for each issue. Some are automatically populated when you create issues
+through the project board itself.
+
+- `Status`: One of the following values:
+    - `NEW`: Default category when an item is added to this board (defined in the workflow)
+    - `CRITICAL`: Indicates that the item is of critical priority. Consider adding `priority/critical-urgent` label too
+    - `FAILING`: Indicates that the issue is about a failing test
+    - `FLAKY`: Indicates that the issue is about a flaking test
+    - `INVESTIGATING`: Default category when an item is re-opened in this board (defined in the workflow)
+    - `OBSERVING`: Indicates that the test mentioned in the issue is under observation
+    - `RESOLVED`: Default category when an item is closed in this board (defined in the workflow)
+    - `PASSING`: Indicates that the test mentioned in the issue is passing
+- `Testgrid Board`: Indicates the board that the failing test is a part of i.e. `master-blocking`, `master-informing` or both
+- `K8s Release`: Signifies the release version this issue is targeting, consider setting the `milestone` label too
+- `Testgrid URL`: The URL of the testgrid job
+- `Date`: Date of addition to the board
+- `CI Signal Member`: The CI Signal member tracking the issue
+- `Slack discussion link`: The slack discussion link in the [#release-ci-signal](https://kubernetes.slack.com/archives/CN0K3TE2C)
+  channel or the SIG/WG channel
+- `view`: Can be one of the following:
+    - `issue-tracking`: **THE DEFAULT VIEW USED, SHOULD BE REMOVED**
+    - `reporting`: **THIS HAS BEEN REPLACED WITH EXCEL SHEET USED FOR TRACKING ASSIGNMENT**
+- `Reporting` : **SHOULD BE REMOVED IMO, JUST UNNECESSARILY COMPLICATING STUFF**
+- `Reporting Status`: **SHOULD BE REMOVED IMO, JUST UNNECESSARILY COMPLICATING STUFF**
```

Comment:
These fields should be removed, IMO. We haven't been using them; they just unnecessarily complicate the task. We maintain an Excel sheet for the CI Signal team where we maintain testgrid observation, meeting assignments, and release cut assignments.

Reply:
Yes, I propose removing this from the schema and creating a new board. I **think** if we remove this from the current board, things might break for previous release boards (like before 1.27), not sure if that is still required. Wdyt?

Reply:
I would recommend to maintain just one board per team. Moving away from excel and entirely to the GitHub project board sounds like a good idea.

Reply:
> Moving away from excel

No, not moving away from excel. We track multiple items in excel sheets like meeting assignments, release cut assignments, primary testgrid observer, team members not available during a time period. These fields do not replace the excel sheet, they just take over 'release cut assignments' portion of excel sheet. We still have to maintain an excel sheet.

Hence, I proposed removing these sections as it is already covered.

Reply:
See [this](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit?usp=sharing) sheet for an example.

Reply:
Hey @leonardpahlke 
Let's conclude this discussion, make the necessary changes and merge this :)

Reply:
> Do what you think works best for the team.

In other words, your proposal sounds good to me.

Reply:
Done, I removed those fields from docs and the board itself.